src/main/java/pl/noip/lolstats/lol/stats/service/SingleChampInfo.java

```diff
@@ -0,0 +1,15 @@
+package pl.noip.lolstats.lol.stats.service;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@ToString
+public class SingleChampInfo {
```

Comment:
Champion not champ. when it's Champion, not Champions it suggests that it's single, so this word is not necessary. Also Info is not required, so rename to Champion.